applications/GeoMechanicsApplication/tests/cpp_tests/test_backward_euler_T_scheme.cpp

```diff
@@ -0,0 +1,56 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_strategies/schemes/backward_euler_T_scheme.hpp"
+#include "spaces/ublas_space.h"
+#include "testing/testing.h"
+
+namespace Kratos::Testing {
+
+using namespace Kratos;
+using SparseSpaceType = UblasSpace<double, CompressedMatrix, Vector>;
+using LocalSpaceType = UblasSpace<double, Matrix, Vector>;
+
+KRATOS_TEST_CASE_IN_SUITE(BackwardEulerScheme_UpdatesVariablesDerivatives_WhenPredictIsCalled,
+                          KratosGeoMechanicsFastSuite)
+{
+    BackwardEulerTScheme<SparseSpaceType, LocalSpaceType> scheme;
+    Model model;
+    auto& model_part = model.CreateModelPart("dummy", 2);
+
+    model_part.AddNodalSolutionStepVariable(TEMPERATURE);
+    model_part.AddNodalSolutionStepVariable(DT_TEMPERATURE);
+
+    const double current_temperature = 1.0;
```

Comment:
It is a matter of taste;  I prefer to use constexpr instead of const.

Reply:
It's not really in my system yet to work with constexpr, but I like the idea of compile time expressions, so I'll try to get it into my system and make the changes, thanks for the suggestion!